docs/source/concepts/identity-and-package-management.rst

```diff
@@ -0,0 +1,169 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Identity and Package Management
+###############################
+
+Since DAML Ledgers enable parties to automate the management of their rights and obligations through smart contract code, they also have to provide party and code management functions.
+Hence, this document addresses:
+
+1. Management of parties' digital identifiers in a DAML Ledger.
+
+2. Distribution of smart contract code between the parties connected to the same DAML Ledger.
+
+The Ledger API provides both a :ref:`party management service <com.digitalasset.ledger.api.v1.PartyService>` and a :ref:`package service <com.digitalasset.ledger.api.v1.PackageService>`.
+The access to these services is usually more restricted compared to the other Ledger API services, as they are part of the administrative API.
+Any implementation of the services is guaranteed to accept inputs and provide outputs of the format specified by these services.
+However, the services' *behavior* -- the relationship between the inputs and outputs that the various parties observe -- is largely implementation dependent.
+The remainder of the document will presents both:
+
+#. the minimal behavioral guarantees for identity and package services across all ledger implementations, and
+
+#. guidelines to understand how the :ref:`ledger's topology <daml-ledger-topologies>` influences the unspecified part of the behavior.
```

Comment:
It would be good if this section clarified
1. who the audience of the document is
2. what questions the seciton will answer.

My take is that the audience is ledger implementors and ledger operators. There seem to be multiple questions:
1. How should I implement party and package managment?
2. Which ledger should I select provided my requirements on party and package management?